### PR TITLE
Logos

### PR DESCRIPTION
**Pull request by mattheu**
I've not done PNG fallbacks yet, but I've added the image files

### CODE REVIEW

File: src/styles/components/_Logo.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
+.Logo {
+
+	overflow: hidden;
+	text-indent: 100%;
+	white-space: nowrap;
```
Comment (sambulance) [inline src/styles/components/_Logo.scss]: Probably want a `display: inline-block` on this

File: src/styles/components/_Logo.scss [hunk 2]
```diff
@@ -0,0 +1,61 @@
+.Logo {
```
Comment (sambulance) [inline src/styles/components/_Logo.scss]: I think we should use prefixes with this being a pattern library - .logo is fairly common and could cause unwanted conflicts

File: src/styles/components/_Logo.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+		height: 20px;
+		&.Logo-red { background-image: url( "../images/logo-tiny-red.svg" ); }
+		&.Logo-white { background-image: url( "../images/logo-tiny-white.svg" ); }
+	}
+
+}
+
+/**
+ * For previewing a list/grid of logos.
+ * This is used in the pattern library.
```
Comment (sambulance) [inline src/styles/components/_Logo.scss]: Should pattern library styles be separate to the pattern library?

File: gulpfile.js [hunk 4]
```diff
@@ -6,6 +6,9 @@ const autoprefixer  = require( 'autoprefixer' );
 const sass          = require( 'gulp-sass' );
 const sassLint      = require( 'gulp-sass-lint' );
 const fileinclude   = require( 'gulp-file-include' );
+const newer         = require( 'gulp-newer' );
+const imagemin      = require( 'gulp-imagemin' );
+const svg2png       = require( 'gulp-svg2png' );
```
Comment (sambulance) [inline gulpfile.js]: Can scrap this now

File: src/styles/components/_Logo.scss (hunk 3 above)
Comment (mattheu) [inline src/styles/components/_Logo.scss]: Yeah I was wondering this. The Swatches could also be treated the same. Lets think about this separately.

File: src/styles/components/_Logo.scss (hunk 2 above)
Comment (mattheu) [inline src/styles/components/_Logo.scss]: Lets leave this for now.

File: src/styles/components/_Logo.scss (hunk 1 above)
Comment (mattheu) [inline src/styles/components/_Logo.scss]: Fixed

File: src/styles/components/_Logo.scss (hunk 3 above)
Comment (sambulance) [inline src/styles/components/_Logo.scss]: Ticket here: https://github.com/humanmade/hm-pattern-library/issues/22

File: src/styles/components/_Logo.scss (hunk 2 above)
Comment (sambulance) [inline src/styles/components/_Logo.scss]: Just wanted to make a note here about our discussion. We have decided not to prefix classes as this is an internal tool and unlikely to conflict with other styles.

Perhaps we should just rename `.logo` to `.hm-logo` incase there are other logos for the project

**Pull request merged**